tests/reconfig/setup.md

```diff
@@ -0,0 +1,96 @@
+# Reconfig tests
+
```

Comment:
To keep things consistent with the other baseline tests, might want to add the test environment here too